leader-election-impl/src/main/java/com/palantir/leader/LeadershipEvents.java

```diff
@@ -40,61 +44,75 @@
     private final Meter leaderPingFailure;
     private final Meter leaderPingTimeout;
     private final Meter leaderPingReturnedFalse;
+    private final Object[] safeArgs;
```

Comment:
I'd prefer a more descriptive name (the idea is along the line of `globallyAddedArgs` though that may be too verbose). Maybe `contextArgs` or something like that?